stockfish.py

```diff
@@ -98,6 +98,9 @@ def set_position(self, moves=None):
             moves = []
         self.__put('position startpos moves %s' %
                    self.__convert_move_list_to_str(moves))
+    
+    def set_fen_position(self, fen):
```

Comment:
please, rename parameter "fen" to "fen_position"

Reply:
please, add this method to README.md and write test on this method

Reply:
Ok - will do soon.